public/products.json

```diff
@@ -0,0 +1,87 @@
+[
```

Comment:
hi 
i would consider below approaches for mocking data:
- using [mockserver](https://github.com/namshi/mockserver/blob/master/README.md) - requires installing a global npm package. but great advantage is that you can put all your fake json data files in to src/__mock__ folder and server it from there with one command
- using [axios-mock-adapter](https://www.npmjs.com/package/axios-mock-adapter) similar to above but more programmatic
- [Fiddler Classic](https://www.telerik.com/download/fiddler) - better to use when debugging real server or for mocking isolated api end points.

Reply:
done